aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDoExample.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box, Notification } from 'grommet';
+
+export const BannerNotificationDoExample = () => (
+  <Box pad="small">
+    <Notification
+      // href="#"
```

Comment:
remove comments? or, I think we could add `onClick` and have it display an alert that says "Clicking on a banner notification could route the user to a notification center for more details."

Reply:
that's what I was debating about if we wanted an `alert` I commented out though since that functionality isn't available through grommet.